<launch>
  <!-- Continous filter to action classification result -->
  <node pkg="decopin_hand" type="continuous_filter.py" name="continuous_filter"
        clear_params="true" output="screen" >
    <remap from="~input" to="/action_classifier/output" />
    <rosparam subst_value="true">
      duration_thre: 0.3
    </rosparam>
  </node>
  <!-- Publish the gesture which the robot hand recognize -->
  <node pkg="decopin_hand" type="gesture-5.l" name="gesture_5"
        output="screen" args="&quot;(main)&quot;"/>
  <!-- gesture classification result -->
  <node name="gesture_classification_result"
          pkg="jsk_perception" type="draw_classification_result.py" >
      <remap from="~input" to="/gesture" />
      <remap from="~input/image" to="/action_classifier/debug/net_input" />
  </node>
  <node pkg="image_view" type="image_view" name="gesture_classification_result_view"  >
    <remap from="image" to="/gesture_classification_result/output" />
  </node>

</launch>
